<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>乐知生活信息平台</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="videoWidget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>560</width>
      <height>331</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <zorder>logoLabel</zorder>
   </widget>
   <widget class="QLabel" name="weatherLabel1">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>330</y>
      <width>141</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="weatherLabel2">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>330</y>
      <width>141</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="weatherLabel3">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>330</y>
      <width>141</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="logoLabel">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>330</y>
      <width>141</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="activityLabel">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>230</y>
      <width>241</width>
      <height>201</height>
     </rect>
    </property>
    <property name="text">
     <string>节气活动</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="newLabel">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>0</y>
      <width>241</width>
      <height>231</height>
     </rect>
    </property>
    <property name="text">
     <string>新闻招聘</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="timeLabel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>430</y>
      <width>221</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>日期时间</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="RollText" name="jinjiNewWidget" native="true">
    <property name="geometry">
     <rect>
      <x>219</x>
      <y>429</y>
      <width>581</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="tsWidget" native="true">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>350</y>
      <width>141</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="RollText" name="danmuWidget" native="true">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>260</y>
      <width>241</width>
      <height>71</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lxLabel">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>230</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>沦陷区</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>140</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>RollText</class>
   <extends>QWidget</extends>
   <header location="global">rolltext.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
